bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -921,7 +921,9 @@ spec:
           value: "4"
         terminationMessagePolicy: FallbackToLogsOnError
       nodeSelector:
+{{- if not .ExternalControlPlane }}
         node-role.kubernetes.io/master: ""
+{{- end }}
```

Comment:
I think it would be clearer to revert this change and the `pkg/network/ovn_kubernetes.go` change entirely, given that the DaemonSet won't actually start if you do set `ExternalControlPlane: true`.

Or maybe better, revert the `ovnkube-master.yaml` change, make `ovn_kubernetes.go` detect `ExternalControlPlane` correctly again, and make `renderOVNKubernetes` return an error if it's true, since it can't properly render the output in that case.

Reply:
Thx, reverted ovnkube-master.yaml and now return an error from `renderOVNKubernetes`